lib/common/signing/sharedkey.js

```diff
@@ -162,7 +162,7 @@ SharedKey.prototype._generateAccountSharedAccessSignature = function(sharedAcces
 /**
 * Signs a request with the Authentication header.
 *
-* @param {WebResource} The webresource to be signed.
+* @param {WebResource} webResource The webresource to be signed.
```

Comment:
Aligning with other parameter.

Reply:
Same in tokensigner.js

---
In reply to: [191098584](https://github.com/Azure/azure-storage-node/pull/453#discussion_r191098584) [](ancestors = 191098584)

Reply:
aligned